# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/tcsh-uclibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- tcsh-6.12.00/config/linux.orig	2005-01-22 13:08:38.000000000 -0800
+++ tcsh-6.12.00/config/linux	2005-01-22 13:08:56.000000000 -0800
@@ -151,6 +151,6 @@
 /* have been moved to config_f.h */
 #include "config_f.h"
 
-#define NLS_CATALOGS
+#undef NLS_CATALOGS
 
 #endif /* _h_config */
